src/form/form.tsx

```diff
@@ -117,7 +117,7 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
             violations: this.props.violations || {},
             initialModel: this.props.model,
             isSubmitting: false,
-            expandedGroupId: this.props.children[0].id,
+            expandedGroupsIds: [this.props.children[0].id],
```

Comment:
How can you be sure that children has always at least one element?

Reply:
Well If you don't provide one section at least, then you are using component incorrectly.